src/attestation_types/mod.rs

```diff
@@ -4,8 +4,8 @@
 //! Section references in further documentation refer to this document.
 //! https://www.intel.com/content/dam/www/public/emea/xe/en/documents/manuals/64-ia-32-architectures-software-developer-vol-3d-part-4-manual.pdf
 
-#![cfg(feature = "std")]
-
+#[cfg(feature = "std")]
 pub mod quote;
+
```

Comment:
Do we need this spacing?

Reply:
Hmm, I suppose it may be more idiomatic without. I will fix shortly.